.code-samples.meilisearch.yaml

```diff
@@ -868,3 +868,11 @@ date_guide_sort_1: |-
       "release_timestamp:desc",
     },
   })
+search_parameter_reference_ranking_score_threshold_1: |-
```

Comment:
You should not copy/paste, but "translate" into go lang as the other examples you can see above 👆

Reply:
> You should not copy/paste, but "translate" into go lang as the other examples you can see above 👆

What is index name for this example?